.travis.yml

```diff
@@ -1,12 +1,30 @@
 language: php
+
+sudo: false
+
 php:
   - 5.5
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 env:
   global:
     secure: Yc+Xohkr/iEUU7FCQuSLXAE9ywNW9g6CfrM1Ki0Hl+fS15F3AXT7dFY8EyCJ4dP1/oI0dBmwrGWrltXV0XWIjGV1Ms3tefCgQpBBAqwT+hImzVP3RbpZW8Iyo2d0VgiDemQF1LWYD/pKu6d8WljTnv5D77NIMdEJjQ0uzeTLWdw=
-install: composer install --dev
```

Comment:
@jpignata 


The `--dev` option is the default, so I removed it. 

See https://getcomposer.org/doc/03-cli.md#install:

>**--dev**: Install packages listed in `require-dev` (this is the default behavior).
>**--no-dev**: Skip installing packages listed in `require-dev`. The autoloader generation skips the `autoload-dev` rules.